### PR TITLE
[SDK-305] Add app id to all web requests

### PR DESCRIPTION
**Pull request by rYuuk**
## Description

- Added app id in core settings and settings window
- App id is sent as header with all requests made from sdk

## How to Test

- Try to check if sent request contains the header

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [ ] Changelog is updated.

[SDK-305]: https://ready-player-me.atlassian.net/browse/SDK-305?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Editor/UI/EditorWindows/SettingsWindow/SettingsEditor.cs [hunk 1]
```diff
@@ -52,7 +52,7 @@ public static void ShowWindow()
         {
             var window = GetWindow<SettingsEditor>();
             window.titleContent = new GUIContent(SETTINGS);
-            window.minSize = new Vector2(500, 580);
+            window.minSize = new Vector2(500, 620);
```
Comment (HarrisonHough) [inline Editor/UI/EditorWindows/SettingsWindow/SettingsEditor.cs:55]: I don't love these inline, hard coded size values

File: Runtime/Utils/WebRequestDispatcher.cs [hunk 2]
```diff
@@ -45,6 +45,8 @@ public class WebRequestDispatcher
                 }
             }
 
+            request.SetRequestHeader("X-APP-ID", CoreSettingsHandler.CoreSettings.AppId);
```
Comment (HarrisonHough) [inline Runtime/Utils/WebRequestDispatcher.cs:48]: X-APP-ID could be moved to a const

**Review by HarrisonHough — APPROVED** [review]
Code looks good. Some small things in comments, not blockers though. Probably wait until Raigo or Max have tested with gateway though

File: Editor/UI/EditorWindows/SettingsWindow/SettingsEditor.cs (hunk 1 above)
Comment (rYuuk) [reply Editor/UI/EditorWindows/SettingsWindow/SettingsEditor.cs:55]: Any suggestions on how to restrict min size for window?

**Review by rk132 — APPROVED** [review]
Tested and looks good

**Pull request merged**